holidayextras/ui-toolkit
# Update to latest version of babel and remove node version

#### What does this PR do? (please provide any background)
- Updates to Babel 6
- Removes the engine dependencies from package.json
- Changes CI tooling to use 5.1 or latest stable version

#### What tests does this PR have?
None

#### How can this be tested?
Delete the UIToolkit from you machine then clone it again. If you don't do this it will unfortunately fail to build.
`npm install`
`npm run docs`
check the docs looks as good and work as well as they do on ui-toolkit.com
You will have no console errors

#### Screenshots / Screencast
None

#### What gif best describes how you feel about this work?
![Babelfish](https://media.giphy.com/media/eDakJwe4NJ9xS/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
